dagger/utilities/dbt_config_parser.py

```diff
@@ -161,16 +161,18 @@ def _generate_dagger_tasks(self, node_name: str) -> List[Dict]:
                 node, follow_external_dependency=True
             )
             dagger_tasks.append(table_task)
-        elif node.get("config", {}).get("materialized") == "ephemeral":
+        elif node.get("config", {}).get("materialized") == "ephemeral" or ((node.get("name").startswith("stg_") or "preparation" in node.get(
```

Comment:
these conditions seem pretty hacky. is there a cleaner way that we could do this?

do you think we could have an approach so that we use a specific tag, which then dagger can use to add dummy tasks.

Reply:
I think at the end of the day, we should remove this parser and change to the jinja way to create the pipelines 🙈